packages/custom-functions-metadata/package.json

```diff
@@ -33,12 +33,12 @@
   },
   "devDependencies": {
     "@types/assert": "^1.4.6",
-    "@types/es6-promise": "^3.3.0",
     "@types/mocha": "^7.0.2",
     "@types/node": "^14.17.2",
     "@types/node-fetch": "^2.5.10",
     "@types/xregexp": "^3.0.30",
     "concurrently": "^6.2.2",
+    "es6-promise": "^4.2.8",
```

Comment:
We don't want to add es6-promise as we did work previously to remove it. We shouldn't need the @types either.

Reply:
The promise polyfill should be coming from babel now.

Reply:
There's a compiler error when I don't have any reference to es6-promise (I'm guessing a compiler option needs to be changed, but I'm not sure of the side effects):

> eslint-plugin-office-addins@1.1.2 build
> rimraf lib && concurrently "tsc -p tsconfig.json"

[0] error TS2688: Cannot find type definition file for 'es6-promise'.
[0]   The file is in the program because:
[0]     Entry point of type library 'es6-promise' specified in compilerOptions
[0] tsc -p tsconfig.json exited with code 2

Reply:
I tried removing 'es6-promise' from the tsconfig.json types array which allowed me to remove the package.  The build and tests work after this so I think it should be OK.